extensions/exposition/source/HTTP/Context.ts

```diff
@@ -27,6 +27,11 @@ export class Context {
     this.timing = new Timing(properties.trace)
     this.debug = properties.debug
 
+    if (this.debug) {
+      console.debug(`${request.method} ${this.url.href}`)
+      console.debug(request.headers)
```

Comment:
Headers must be sanitised before being displayed on console as it may contain some sensitive information and console output goes to persistent logs in many cases

Reply:
What sensitive information are we talking about? Authorization header, I suppose, what else?

Reply:
fixed